rpc/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "tendermint-rpc"
+version = "0.1.0"
+authors = ["Alexander Simmerl <a.simmerl@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+async-tungstenite = {version="0.5", features = ["tokio-runtime"]}
+bytes = "0.5"
+futures = "0.3"
+getrandom = "0.1"
+http = "0.2"
+hyper = "0.13"
+serde = { version = "1", features = [ "derive" ] }
+serde_bytes = "0.11"
+serde_json = "1"
+tendermint = { version = "0.13.0", path = "../tendermint" }
```

Comment:
I'm always sceptical of circular dependencies (even if it's just a dev-dependency). Are we confident that this does cannot have weird side-effects?

Reply:
I would not expect any, but truth be told this was the solution with the least amount of opinionated changes. Would like to get rid of this circle as well, but afraid this goes too far for this PR.

Reply:
I can't think of any problems either 🍀 🤞